types/altair/sync_committee.yaml

```diff
@@ -67,43 +63,35 @@ Altair:
     required: [slot, beacon_block_root, subcommittee_index, aggregation_bits, signature]
     properties:
       slot:
-        allOf:
-          - $ref: "../primitive.yaml#/Uint64"
-          - description: "The slot at which the validator is providing a sync committee contribution."
+        $ref: "../primitive.yaml#/Uint64"
+        description: "The slot at which the validator is providing a sync committee contribution."
       beacon_block_root:
-        allOf:
-          - $ref: '../primitive.yaml#/Root'
-          - description: "Block root for this contribution."
+        $ref: '../primitive.yaml#/Root'
+        description: "Block root for this contribution."
       subcommittee_index:
-        allOf:
-          - $ref: '../primitive.yaml#/Uint64'
-          - description: 'The index of the subcommittee that the contribution pertains to.'
+        $ref: '../primitive.yaml#/Uint64'
+        description: 'The index of the subcommittee that the contribution pertains to.'
       aggregation_bits:
-        allOf:
-          - description: 'A bit is set if a signature from the validator at the corresponding index in the subcommittee is present in the aggregate `signature`.'
-          - $ref: "../primitive.yaml#/Bitvector"
-          - example: "0xffffffffffffffffffffffffffffffff"
+        description: 'A bit is set if a signature from the validator at the corresponding index in the subcommittee is present in the aggregate `signature`.'
+        $ref: "../primitive.yaml#/Bitvector"
+        example: "0xffffffffffffffffffffffffffffffff"
```

Comment:
does not follow ordering discussed in https://github.com/ethereum/beacon-APIs/pull/407#discussion_r1450450306

Reply:
The idea is to keep the order unchanged in this PR (should be the case now) and have another one introducing a consistent order with associated CI and command to fix it.